apps/client/src/components/card/card.tsx

```diff
@@ -1,79 +1,36 @@
-import { CSSProperties } from 'react';
 import styles from './card.module.css';
-import { Divider } from '..';
-import { styled, Tooltip, tooltipClasses, TooltipProps } from '@mui/material';
+import { PropsWithChildren } from 'react';
 
 type Props = {
   title: string | undefined;
   stringBorder?: string;
   description?: string;
-  descriptionStyle?: CSSProperties;
   date?: Date;
-  height?: string;
-  fluids?: string;
-  solids?: string;
-};
+} & PropsWithChildren;
 export const Card: React.FC<Props> = ({
   title,
   description,
   stringBorder,
-  descriptionStyle,
   date,
-  height,
-  fluids,
-  solids,
+  children,
 }) => {
   const style: Record<string, string> = {};
   if (stringBorder) {
     style['border'] = stringBorder;
   }
-  if (height) {
-    style['height'] = height;
-  }
-  const CustomWidthTooltip = styled(({ className, ...props }: TooltipProps) => (
-    <Tooltip {...props} classes={{ popper: className }} />
-  ))({
-    [`& .${tooltipClasses.tooltip}`]: {
-      width: '31.25rem',
-    },
-  });
+
   const charOfNewDate = 10;
   const formattedDate = date
     ? date.toString().substring(0, charOfNewDate)
     : null;
   return (
-    <>
-      {fluids && solids ? (
-        <CustomWidthTooltip
-          title={
-            <div className={styles.tooltip}>
-              <div className={styles.tooltipheader}>:משקאות</div>
-              <Divider />
-              <div className={styles.tooltiptext}>{fluids}</div>
-              <div className={styles.tooltipheader}>:מאכלים</div>
-              <Divider />
-              <div className={styles.tooltiptext}>{solids}</div>
-            </div>
-          }
-          placement="top"
-        >
-          <div className={styles.card} style={style}>
-            <div className={styles.heading}>{title}</div>
-            <div className={styles.desc} style={descriptionStyle}>
-              {description}
-            </div>
-            {date && <div>{formattedDate}</div>}
-          </div>
-        </CustomWidthTooltip>
-      ) : (
-        <div className={styles.card} style={style}>
-          <div className={styles.heading}>{title}</div>
-          <div className={styles.desc} style={descriptionStyle}>
-            {description}
-          </div>
-          {date && <div className={styles.date}>{formattedDate}</div>}
-        </div>
-      )}
-    </>
+    <div className={styles.card} style={style}>
+      <div className={styles.cardHeader}>
```

Comment:
What is this Header for? as it is right now you are placing all the children inside the heading